platform/o.n.bootstrap/apichanges.xml

```diff
@@ -29,6 +29,21 @@
   </apidefs>
 
   <changes>
+    <change id="bootstrap.filter">
+        <api name="launcher"/>
+        <summary>Can use classloader to check access to bootstrap classes</summary>
+        <version major="2" minor="80"/>
+        <date day="5" month="9" year="2018"/>
+        <author login="sdedic"/>
+        <compatibility addition="yes" binary="compatible" semantic="compatible" deprecation="yes"/>
```

Comment:
I guess the semantic is "incompatible".

I believe we should do the fix. It leads towards more robust system. However I'd use the API change note to stress, that the change isn't really compatible. For example summary could be "Hiding non-exported implementation classes" and description could give examples of classes that are no longer going to be visible in post 9.0 version.

This is an advice.